styleguide/source/assets/scss/03-organisms/_four-up-teaser.scss

```diff
@@ -12,28 +14,29 @@
   }
 
   a {
-    @include font-size($p-font-sizes);
+    @extend %ama__type--small;
```

Comment:
Does this mixin include sizing for the line height? It seems like the font got smaller but the line height didn't scale appropriately.

Reply:
all line heights are governed by the font-variables css which gets fed into the font_mixins